py3status/modules/kdeconnector.py

```diff
@@ -119,6 +119,17 @@ def _get_device_id(self, bus):
 
         return None
 
+    def _get_isTrusted(self):
+        try:
+            # New method which replaced 'isPaired' in version 1.0
+            return self._dev.isTrusted
+        except Exception:
+            try:
+                # Deprecated
+                return self._dev.isPaired
+            except Exception:
+                return None
```

Comment:
when you do `device['isTrusted']()` later this will fail.

maybe do the function call inside this function and just return True/False